test/init.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = require('should');
+
+var DataSource = require('loopback-datasource-juggler').DataSource;
+
+
+
+var config;
+
+config = {
+    connector: "loopback-connector-cos",
+    name: "cloud-object-storage",
+    endpoint: process.env.COS_ENDPOINT,
+    apiKeyId: process.env.API_KEY_ID,
```

Comment:
Can we prefix this property with `COS_`, to distinguish it from other possible API KEY IDs and also for consistency with other env properties? Unless API key can be shared by more things than just COS?